src/ui/frmpreferences.ui

```diff
@@ -114,6 +114,13 @@
           </widget>
          </item>
          <item>
+          <widget class="QCheckBox" name="chkRememberSession">
+           <property name="text">
+            <string>Remember openened tabs when closing Notepadqq</string>
```

Comment:
"openened"... I'm pointing out these trivial issues now because otherwise I'm sure I'm going to forget about them.